grid-client/integration_tests/batch_gateway_name_test.go

```diff
@@ -43,20 +43,13 @@ func TestBatchGatewayNameDeployment(t *testing.T) {
 	nodeID1 := uint32(nodes[0].NodeID)
 	nodeID2 := uint32(nodes[1].NodeID)
 
-	network := generateBasicNetwork([]uint32{nodeID1, nodeID2})
-
-	vm := workloads.VM{
-		Name:        "vm",
-		NodeID:      nodeID1,
-		NetworkName: network.Name,
-		CPU:         minCPU,
-		MemoryMB:    minMemory * 1024,
-		Planetary:   true,
-		Flist:       "https://hub.grid.tf/tf-official-apps/base:latest.flist",
-		Entrypoint:  "/sbin/zinit init",
-		EnvVars: map[string]string{
-			"SSH_KEY": publicKey,
-		},
+	network, err := generateBasicNetwork([]uint32{nodeID1, nodeID2})
+	if err != nil {
```

Comment:
if network returned error because it couldn't create mycelium key for a node then will not be able to connect through mycelium, so the remote run or the check connection will fail. it depends on network Creation right

Reply:
if we had an error we need to be notified as something changed in the pkg for example, It shouldn't have an error